src/main/java/com/plugish/woominecraft/WooMinecraft.java

```diff
@@ -175,12 +175,15 @@ public boolean check() throws Exception {
 				continue;
 			}
 
-			// If the user isn't in a white-listed world, commands will not run here.
-			if ( !getConfig().getStringList( "whitelist-worlds" ).contains( player.getWorld().getName() ) ) {
+			//Check for enabled world whitelisting in config
+			if(getConfig().getBoolean("enable-world-whitelist")){
+				// If the user isn't in a white-listed world, commands will not run here.
+				if ( !getConfig().getStringList( "whitelist-worlds" ).contains( player.getWorld().getName() ) ) {

```

Comment:
Work needs to be done on this, I agree, but overall I'm almost 100% positive there's a better way than stringing this together.